civihr_employee_portal/templates/civihr-employee-portal-civihr-report-custom.tpl.php

```diff
@@ -6,25 +6,27 @@
 </div>
 <?php endif; ?>
 
-<ul class="nav nav-tabs nav-justified report-tabs">
+<ul class="nav nav-tabs nav-justified report-tabs panel chr_tabs__header">
 <?php if (!empty($tableUrl)): ?>
-    <li role="presentation" class="active"><a class="btn btn-default" data-tab="data">Data</a></li>
+    <li role="presentation" class="chr_tabs__header-item active"><a class="btn btn-default" data-tab="data">Data</a></li>
 <?php endif; ?>
 <?php if (!empty($jsonUrl)): ?>
-    <li role="presentation"><a class="btn btn-default" data-tab="pivot-table">Pivot Table</a></li>
+    <li role="presentation" class="chr_tabs__header-item"><a class="btn btn-default" data-tab="pivot-table">Pivot Table</a></li>
 <?php endif; ?>
 <?php if (!empty($jsonUrl)): ?>
-    <li role="presentation"><a class="btn btn-default" data-tab="orb-pivot-table">Orb Pivot Table</a></li>
+    <li role="presentation" class="chr_tabs__header-item"><a class="btn btn-default" data-tab="orb-pivot-table">Orb Pivot Table</a></li>
 <?php endif; ?>
 </ul>
 
-<div class="report-content">
+<div class="report-content panel">
 <?php if (!empty($tableUrl)): ?>
     <div class="report-block data">
         <h4>Data</h4>
         <div id="reportTable"><?php print $table; ?></div>
 <?php if (!empty($exportUrl)): ?>
-        <a href="<?php print $exportUrl; ?>" id="export-report" class="btn btn-primary btn-default">Export</a>
+        <div class="export-button-wrap">
```

Comment:
This blue-button-on-bottom-right-corner thing is already covered by the {{.chr_panel}} component and in particular by the {{.chr_panel__footer}} element. You can see it in action in most of the panels in the SSP dashboard. Please take a look if you can reuse it and scrap this custom element altogether